spec/controller/users_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+include SessionsHelper
+RSpec.describe UsersController, type: :controller do
+  let(:user_1) {FactoryBot.create :user}
```

Comment:
let có tác dụng gì em

Reply:
khai báo 1 biến để sử dụng trong mỗi test case, nhưng let sẽ khởi tạo giá trị chỉ khi gọi đến nó và nếu trong 1 it gọi nó nhiều lân thì nó chỉ sử dụng giá trị đầu tiên được gọi trong it thôi ạ.